devlab/tests/real_env_conf.py

```diff
@@ -0,0 +1,135 @@
+        'SHym3kZtZPSTgT6GIGoWA1+QHlhx5kiMVEN+YRZF vagrant'}
+]
+
+# Images to create/delete
+images = [
+    {'name': 'image1', 'copy_from': 'http://download.cirros-cloud.net/0.3.3/ci'
+                                    'rros-0.3.3-x86_64-disk.img',
+     'is_public': True},
+    {'name': 'image2', 'copy_from': 'http://download.cirros-cloud.net/0.3.3/ci'
+                                    'rros-0.3.3-x86_64-disk.img',
+     'container_format': 'bare', 'disk_format': 'qcow2', 'is_public': False}
+]
+
+# Flavors to create/delete
+flavors = [
+    {'name': 'm1.tiny'}
+    # {'name': 'flavorname1', 'disk': '7', 'ram': '64', 'vcpus': '1'},
+    # Disabled for now, but in the future we need to generate non-pubic flavors
+    # {'name': 'flavorname3', 'disk': '10', 'ram': '32', 'vcpus': '1',
+    #  'is_public': False},
+    # {'name': 'flavorname2', 'disk': '5', 'ram': '48', 'vcpus': '2'}
+]
+
+# Security groups to create/delete
+security_groups = [{'security_groups': [
+    {'name': 'sg11',
+     'description': 'Blah blah group',
+     'rules': [{'ip_protocol': 'icmp',
+                'from_port': '0',
+                'to_port': '255',
+                'cidr': '0.0.0.0/0'},
+               {'ip_protocol': 'tcp',
+                'from_port': '80',
+                'to_port': '80',
+                'cidr': '0.0.0.0/0'}]},
+    {'name': 'sg12',
+     'description': 'Blah blah group2'}]}]
+
+
+# Networks to create/delete
+# Connected to tenants
+ext_net = {'name': 'shared_net', 'admin_state_up': True, 'shared': True,
+           'router:external': True}
+
+networks = [
+    {'name': 'mynetwork1', 'admin_state_up': True},
+]
+
+# Subnets to create/delete
+ext_subnet = {'cidr': '172.18.10.0/24', 'ip_version': 4}
+
+subnets = [
+    {'cidr': '10.4.2.0/24', 'ip_version': 4},
+]
+
+# VM's to create/delete
+vms = [
+    {'name': 'server1', 'image': 'image1', 'flavor': 'm1.tiny'},
+    {'name': 'server2', 'image': 'image2', 'flavor': 'm1.tiny'},
+    {'name': 'server3', 'image': 'image1', 'flavor': 'm1.tiny'},
+    {'name': 'server4', 'image': 'image2', 'flavor': 'm1.tiny'},
+    {'name': 'server5', 'image': 'image1', 'flavor': 'm1.tiny'},
+    {'name': 'server6', 'image': 'image1', 'flavor': 'm1.tiny'}
+]
+
+routers = [
+    {
+        'router': {
+            'name': 'ext_router',
+            'external_gateway_info': {
+                'network_id': 'shared_net'},
+            'admin_state_up': True}}
+]
+
+# VM's snapshots to create/delete
+snapshots = [
+    {'server': 'server2', 'image_name': 'asdasd'}
+]
+
+# Cinder images to create/delete
+cinder_volumes = [
+    {'name': 'cinder_volume1', 'size': 1},
+    {'name': 'cinder_volume2', 'size': 1,
+     'server_to_attach': 'server2', 'device': '/dev/vdb'}
+]
+
+# Cinder snapshots to create/delete
+cinder_snapshots = [
+    # Commented because of unimplemented error in nfs driver for grizzly.
+    # {'display_name': 'snapsh1', 'volume_id': 'cinder_volume1'}
```

Comment:
We should generate cinder snapshots in prod, this was commented only for devenv where NFS driver does not support snapshots.